scripts/collect-strains-sars-cov-2.js

```diff
@@ -0,0 +1,75 @@
+#!/usr/bin/env node
+
+const AWS = require("aws-sdk");
+const fetch = require("node-fetch");
+const fs = require('fs');
+
+const collectStrainNames = (name, json) => {
+  const strains = [];
+  const recurse = (node) => {
+    if (node.children) {
+      for (let i=0; i<node.children.length; i++) recurse(node.children[i]);
+    } else {
+      strains.push(node.name);
+    }
+  };
+  try {
+    recurse(json.tree);
+  } catch (err) {
+    console.log(`Error collecting strain names for ${name}`);
+    return [];
+  }
+  return strains;
+};
+
+(async () => {
+  const S3 = new AWS.S3();
+  const BUCKET = `nextstrain-staging`;
+  const NEXTSTRAIN_URL_PREFIX = `https://nextstrain.org/staging/`;
+  let s3Objects;
+  try {
+    s3Objects = await S3.listObjectsV2({Bucket: BUCKET}).promise();
+  } catch (err) {
+    console.log("Error listing objects via the S3 API -- were credentials correctly set?");
```

Comment:
Should we add something to docs for open source contributors about how unless you have nextstrain bucket AWS creds you should run `npm run build static && npm run build auspice` since `npm run build` will here crash otherwise (as it did for me since I have none set up for nextstrain right now)?

Reply:
Good idea -- documentation added and exit codes modified (which were one reason TravisCI was failing)